src/core/Akka.Persistence/Persistent.cs

```diff
@@ -145,13 +96,15 @@ public AtomicWrite(IImmutableList<IPersistentRepresentation> payload)
         }
 
         /// <summary>
-        /// TBD
+        /// This persistent message's payload.
         /// </summary>
         public object Payload { get; private set; }
+
         /// <summary>
         /// TBD
         /// </summary>
-        public IActorRef Sender { get; private set; }
+        public IActorRef Sender { get; }
```

Comment:
Considering you added XML comments for lots of other part. You might as well do these as well ;)